tests/strategy/test_wrapping_strategy.py

```diff
@@ -0,0 +1,27 @@
+from unittest import mock
+from yapapi.strategy import WrappingMarketStrategy
+
+
+class MyBaseStrategy(mock.Mock):
+    some_other_value = 123
+
+
+class MyStrategy(WrappingMarketStrategy):
+    some_value = 456
+
+    def some_method(self):
+        return True
+
+
+def test_wrapping_strategy():
+    base_strategy = MyBaseStrategy()
+    strategy = MyStrategy(base_strategy)
+
+    assert strategy.some_value == MyStrategy.some_value
+    assert strategy.some_other_value == MyBaseStrategy.some_other_value
+
+    strategy.some_method()
+    assert not base_strategy.some_method.called
```

Comment:
There's no `some_method` on `base_strategy`, I don't understand this check.

Reply:
`base_strategy` is a mock - so any properties referred-to are "there" if checked for ... this check just verifies that this call did _not_ pass through to the base_strategy